subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoPluginIntegrationTest.groovy

```diff
@@ -364,24 +364,25 @@ Artifacts
                 visible = false
                 canBeResolved = true
                 canBeConsumed = false
-                extendsFrom(configurations.implementation)
-                attributes {
-                    attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category, Category.VERIFICATION))
-                    attribute(VerificationType.VERIFICATION_TYPE_ATTRIBUTE, objects.named(VerificationType.class, VerificationType.JACOCO_RESULTS))
-                }
             }
-
+            coverageDataConfig.dependencies.add(project.dependencies.create(project))
```

Comment:
Like I say below, I think we should delete these "via Dependency Management" tests and leave this kind of testing up to the aggregation plugin tests.